public/static/docs/user-guide/contributing/docs.md

```diff
@@ -154,3 +154,20 @@ pre-commit hook that is integrated when `yarn` installs the project dependencies
 > Check out the `.md` source code of any command reference to get a better idea,
 > for example in
 > [this very file](https://raw.githubusercontent.com/iterative/dvc.org/master/public/static/docs/user-guide/contributing/docs.md).
+
+## General language guidelines
```

Comment:
Would it make sense to formalize and start giving examples in terms how we choose terms - being human-friendly first, consistent (to a reason), etc?

Reply:
OK good idea. I'm not totally sure what you have in mind but check out this paragraph I added in 6347a36f:

```markdown
We prefer human-friendly language than exact jargon, but we do seek preciseness,
even if using simple, general terms. Our
[command-references](/doc/command-reference) and some
[user-guides](/doc/user-guide) are the most technical documents we have, where
specialized terminology is used the most, but even there, often we use
expandable sections for complex implementation details.
```

Also not sure what you meant exactly by "start giving examples" though.

Reply:
`preciseness` sounds strange - correctness? not sure how to better put it


example: `prefer commit over revision whenever possible`
example: options in commands - start with the essence first, simple language, clarifications, edge cases, precision after ...

Reply:
Updated to:

```markdown
We prefer human-friendly language than exact jargon, as long as it's correct,
even if using general terminology. Example: avoid Git jargon such as _revision_
or _reference_, preferring the more basic concept _commit_.

The [command-reference](/doc/command-reference) contains some of our most
technical documents where specialized language is used the most, but even there,
we use expandable sections for complex implementation details.

Start by writing the essence in simple terms, and complete it with
clarifications, edge cases, or other precisions in a separate iteration.
```

Better? Too long now?

Reply:
looks great! :)